project/OSGi.scala

```diff
@@ -16,7 +16,7 @@ object OSGi {
   // in the .../bundles directory which makes testing locally published artifacts
   // a pain. Create bundles but publish them to the normal .../jars directory.
   def osgiSettings = defaultOsgiSettings ++ Seq(
-    packagedArtifact in (Compile, packageBin) := ((artifact in (Compile, packageBin)).value, OsgiKeys.bundle.value),
+    Compile / packageBin := OsgiKeys.bundle.value,
```

Comment:
This doesn't look equivalent to me, but it's ok?

Reply:
Indeed: by default `packagedArtifact` is `(artifact.value, packageBin.value)`. This change makes sure the default `packageBin` is replaced with the OSGi bundle instead of only the `packagedArtifact`, so any problems will be already visible with `package` and not only on `publishLocal`.

I thought this would make things less confusing. If it has unintended consequences it can be reverted.

Reply:
Thanks for clarifying!